Source/SwiftLintBuiltInRules/Rules/Performance/EmptyCountRule.swift

```diff
@@ -59,6 +97,67 @@ private extension EmptyCountRule {
             }
         }
     }
+
+    final class Rewriter: ViolationsSyntaxRewriter {
+        private let operators: Set = ["==", "!=", ">", ">=", "<", "<="]
+        private let configuration: EmptyCountConfiguration
+
+        init(configuration: EmptyCountConfiguration,
+             locationConverter: SourceLocationConverter,
+             disabledRegions: [SourceRange]) {
+            self.configuration = configuration
+            super.init(locationConverter: locationConverter, disabledRegions: disabledRegions)
+        }
+
+        override func visit(_ node: SequenceExprSyntax) -> ExprSyntax {
+            guard let folded = try? OperatorTable.standardOperators.foldSingle(node) else { return super.visit(node) }
+
+            if let infix = folded.as(InfixOperatorExprSyntax.self) {
+                return visit(infix)
+            } else {
+                return super.visit(folded)
+            }
+        }
+
+        override func visit(_ node: InfixOperatorExprSyntax) -> ExprSyntax {
+            guard let operatorNode = node.operator.as(BinaryOperatorExprSyntax.self),
+                  let binaryOperator = operatorNode.operator.binaryOperator,
+                  operators.contains(binaryOperator) else {
+                return super.visit(node)
+            }
+
+            if let (count, position) = node.countNodeAndPosition(onlyAfterDot: configuration.onlyAfterDot) {
+                let newNode: ExprSyntax? = if let count = count.as(MemberAccessExprSyntax.self) {
+                    count.with(\.declName.baseName, "isEmpty").trimmed.as(ExprSyntax.self)
+                } else if let count = count.as(DeclReferenceExprSyntax.self) {
+                    count.with(\.baseName, "isEmpty").trimmed.as(ExprSyntax.self)
+                } else {
+                    nil
+                }
```

Comment:
While I'm not agains the new `if` expressions, I'm still unsure how to format them in a clear and readable way. Either

```swift
let x = if ... {
            a
        } else {
            b
        }
```
or
```swift
let x =
    if ... {
        a
    } else {
        b
    }
```
look a bit better, though.

Reply:
Fixed at 4b81a1d. (I think I'm a fan of the latter.)